test/rockpaperscissors.js

```diff
@@ -0,0 +1,772 @@
+            await rcp.movePlayerTwo(gameKey, playerTwoMoveHash, {from: playerTwo, value: toBN(10)});
+            const txObj = await rcp.revealPlayerOne(playerOneSecretBytes32, SCISSORS, {from: playerOne});
+
+            const timestamp = await getBlockTimeStamp(txObj);
+            const expiryDate = timestamp + DAY_24_HOUR_IN_SECS;
+
+            await truffleAssert.eventEmitted(txObj, 'PlayerOneReveals', (ev) => {
+
+                return  ev.player === playerOne &&
+                        ev.gameKey === gameKey &&
+                        ev.move.toString(10) === "3" &&
+                        ev.expiryDate.toString(10) === expiryDate.toString(10)
+            },'PlayerOneReveals event is emitted');
+
+            const game = await rcp.games(gameKey);
+            assert.strictEqual(game.playerOneMove.toString(10), SCISSORS.toString(10));
+            assert.strictEqual(game.step.toString(10), PLAYER_ONE_REVEAL);
+            assert.strictEqual(game.expiryDate.toString(10), expiryDate.toString(10));
+        });
+
+
+        it("Player two reveals and player two wins", async () => {
+
+            gameKey = await rcp.createPlayerOneMoveHash(playerOne, playerOneSecretBytes32, ROCK);
+            await rcp.movePlayerOne(gameKey, toBN(10), {from: playerOne, value: toBN(10)});
+            playerTwoMoveHash = await rcp.createPlayerTwoMoveHash(playerTwo, gameKey, playerTwoSecretBytes32, PAPER);
+            await rcp.movePlayerTwo(gameKey, playerTwoMoveHash, {from: playerTwo, value: toBN(10)});
+            await rcp.revealPlayerOne(playerOneSecretBytes32, ROCK, {from: playerOne});
+            const txObj = await rcp.revealPlayerTwo(gameKey, playerTwoSecretBytes32, PAPER, {from: playerTwo});
+
+            await truffleAssert.eventNotEmitted(txObj, 'DrawRefund');
+
+            await truffleAssert.eventEmitted(txObj, 'PlayerTwoReveals', (ev) => {
+
+                return  ev.player === playerTwo &&
+                        ev.gameKey === gameKey &&
+                        ev.move.toString(10) === "2"
+            },'PlayerTwoReveals event is emitted');
+
+            await truffleAssert.eventEmitted(txObj, 'GamePayment', (ev) => {
+
+                return  ev.player === playerTwo &&
+                        ev.gameKey === gameKey &&
+                        ev.amount.toString(10) === "20"
+            },'GamePayment event is emitted');
+
+            const game = await rcp.games(gameKey);
+            assert.strictEqual(game.stake.toString(10), INIT);
+            assert.strictEqual(game.playerOne, playerOne);
+            assert.strictEqual(game.playerTwo, ZERO_ADDRESS);
+            assert.strictEqual(game.playerTwoMoveHash, ZERO_BYTES_32);
+            assert.strictEqual(game.playerOneMove.toString(10), "0");
+            assert.strictEqual(game.expiryDate.toString(10), "0");
+            assert.strictEqual(game.step.toString(10), "0");
```

Comment:
You could gain by creating a separate function that does these assertions on a game. You would pass it the expected game.